src/shared/sam/cli/samCliDeploy.ts

```diff
@@ -40,8 +53,8 @@ export class SamCliDeployInvocation {
         console.error(`stderr: ${stderr}`)
         console.error(`stdout: ${stdout}`)
 
-        const err =
-            new Error(`sam deploy encountered an error: ${error && error.message ? error.message : stderr || stdout}`)
+        const message = error && error.message ? error.message : stderr || stdout
```

Comment:
Our SAM error messages are starting to diverge. Should we stick to this format or change to something else? FWIW, I did the following for `sam --info` in #388 (this also got rid of the `console.error` statements. https://github.com/aws/aws-toolkit-vscode/blob/d22827ada1b60028398428a3749cc174fc0b4bcb/src/shared/sam/cli/samCliInvoker.ts#L133

Reply:
Yeah, we should have a common utility instead of repeating this everywhere. Let's do that (and decide on the format) in a separate change.